frontend/src/components/rota/index.tsx

```diff
@@ -27,7 +28,7 @@ const DUMMY_EVENTS = [
   },
 ];
 
-const tooltipFunction = (info: any) => {
+const tooltipFunction = (info: { event: EventApi; el: HTMLElement }) => {
```

Comment:
Nice! Is there an easy way of looking up these types? I tried looking for them but settled for `any` in the end...

Reply:
I just got those types from VSC. Unfortunatley `info` itself is not a named typed. We'd need to extract from the function so for me its cleaner to just take the parts we need.

![image](https://user-images.githubusercontent.com/2536293/83873127-19698700-a72b-11ea-8c01-5182abfdb2ab.png)